rules/library.bzl

```diff
@@ -304,11 +304,12 @@ def _xcframework_slice(*, xcframework_name, identifier, platform, platform_varia
         # We need the path to the top-level slice directory so we can find headers, modulemaps and swiftmodules.
         slice_dir = paths.dirname(path)
         import_headers, import_module_map, import_swiftmodules = _xcframework_slice_imports(slice_dir)
+        includes = [paths.dirname(f) for f in import_headers]
         native.objc_import(
             name = resolved_target_name,
             archives = [path],
             hdrs = import_headers,
-            includes = ["%s/Headers" % slice_dir],
+            includes = includes,
```

Comment:
Assuming that the pattern here is always going to be `%s/Headers` is error prone. The test case that was already checked into `rules_ios` (`ObjcLib` under `xcframeworks` tests) is one example. The header is located under `Headers/ObjcLib`.